tensorflow/python/ops/image_ops_test.py

```diff
@@ -551,21 +567,31 @@ def testError(self):
 class PadToBoundingBoxTest(test_util.TensorFlowTestCase):
```

Comment:
Can you add a test that checks that it is possible to fail these various checks? I know that should have existed before but now we need it even more.